### PR TITLE
export-excel

### PR DESCRIPTION
**Pull request by huythuy2495**

### CODE REVIEW

**Comment by huythuy2495** [comment]
ready

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -1,10 +1,16 @@
 module Admin
   class OrdersController < AdminBaseController
     before_action :load_order, only: :update
+    
     def index
       @orders = Order.list_order
+      respond_to do |format|
+        format.html
+        format.xls { send_data @orders.to_csv(col_sep: "\t") }
+        format.xls
```
Comment (huuphuc93) [inline app/controllers/admin/orders_controller.rb]: sao lại 2 format.xls

File: app/views/admin/orders/index.html.erb [hunk 2]
```diff
@@ -22,6 +22,9 @@
           </tbody>
         </table>
       </div>
+        <p> 
+          <%= link_to "Download Excel", admin_orders_path(format: "xls") %>
```
Comment (huuphuc93) [inline app/views/admin/orders/index.html.erb]: I18n

**Comment by huuphuc93** [comment]
commented

**Comment by huythuy2495** [comment]
ready

**Comment by huuphuc93** [comment]
merged

**Pull request merged**